crt_portal/cts_forms/views.py

```diff
@@ -980,6 +980,12 @@ def get_report_data(self, requested_reports):
             })
         return data
 
+    def reconstruct_id_args(self, ids):
+        id_args = ''
+        for id in ids:
+            id_args += f'&id={id}'
+        return id_args
```

Comment:
A _fun_ ctional way to do this:

```suggestion
        return ''.join([f'&id={id}' for id in ids])
```